pandas/core/generic.py

```diff
@@ -165,12 +166,15 @@ def _validate_dtype(self, dtype):
 
         if dtype is not None:
             dtype = _coerce_to_dtype(dtype)
```

Comment:
I think can just directly use ``pandas._dtype`` here. In fact I *think* you can replace *all* usages of ``_coerce_to_dtype`` with ``pandas_dtype``. (you might have to back off some if the tests don't pass), but can investigate those.